packages/oauth/src/stores/file-store.ts

```diff
@@ -30,7 +30,13 @@ export default class FileInstallationStore implements InstallationStore {
     const installationDir = this.getInstallationDir(enterprise?.id, team?.id);
 
     if (logger !== undefined) {
-      logger.info(`Storing installation in ${installationDir} for ${JSON.stringify({ enterprise, team, user })}`);
+      const dataForLogging = {
+        enterprise,
+        team,
+        // user object can include token values
+        user: { id: user.id },
```

Comment:
While doing tests this time, I found that this info-level logging can print user tokens